cms/djangoapps/contentstore/tasks.py

```diff
@@ -125,3 +126,17 @@ def push_course_update_task(course_key_string, course_subscription_id, course_di
     # TODO Use edx-notifications library instead (MA-638).
     from .push_notification import send_push_course_update
     send_push_course_update(course_key_string, course_subscription_id, course_display_name)
+
+
+@task()
+def async_export_to_git(course_module, user=None):
+    """
+    Exports a course module to Git.
+    """
+    try:
+        export_to_git(course_module.id, course_module.giturl, user=user)
+        LOGGER.debug('Exported course content to git: %s', course_module.id)
```

Comment:
i'm following the convention of the rest of the tasks in this module. the other tasks use `LOGGER.debug` to log messages. examples: https://github.com/mitodl/edx-platform/pull/13/files#diff-e5cb44a7b9f87fcd4099b29724da5d24R105 and https://github.com/mitodl/edx-platform/pull/13/files#diff-e5cb44a7b9f87fcd4099b29724da5d24R118